README.md

```diff
@@ -84,3 +84,9 @@ Usually, the `request-language` should not be set, except for the following case
 * If both of the following conditions are met, the `request-language` should be set to the language from the first condition:
     1. The company in question only addresses consumers in one language.
     2. The `relevant-countries` for the company include ones where this language is not spoken.
+
+## Relevant Countries
+
+`relevant-countries` is a list of countries the record is relevant for. The countries can either be an ISO 3166-1 alpha-2 country code or 'all'. If a company has branches in different countries with unique contact data, separate records should be created for those. If a company branch offers its services in English and isn't explicitly restricted to certain countries, 'all' should be specified.
```

Comment:
Not a fan of duplicating the text from the schema here. That seems like a recipe for lots of unnecessary work or (more likely) forgetting to update the text in one of the places, leading to inconsistent documentation.

The other elements don't have extensive descriptions of what they are, either. Mostly just additional restrictions/clarifications to the schema.